README.md

```diff
@@ -15,7 +15,7 @@ $ git clone git@github.com:wildfly-security/wildfly-elytron.git
 Setup the JBoss Maven Repository
 --------------------------------
 
-To use dependencies from JBoss.org, you need to add the JBoss Maven Repositories to your Maven settings.xml. For details see http://community.jboss.org/wiki/MavenGettingStarted-Users
+To use dependencies from JBoss.org, you need to add the JBoss Maven Repositories to your Maven settings.xml. For details see [MavenGettingStarted-Users](https://developer.jboss.org/docs/DOC-15169)
```

Comment:
@RanabirChakraborty  Thanks for the PR! You can use spaces in [MavenGettingStarted-Users] for readability.

Reply:
Thanks! did the changes.

Reply:
Thank you